flashinfer_bench/bench/evaluators/dsa_sparse_attention.py

```
@@ -0,0 +1,128 @@
+                    torch.cuda.synchronize(device)
+                    out = _flexible_normalize(result, device)
+            except Exception:
+                traceback.print_exc()
+                return None, make_eval(
+                    status=EvaluationStatus.RUNTIME_ERROR, device=device, log_path=log_path
+                )
+
+            ref_out = ref_outputs[trial]
+            n_compare = min(len(out), len(ref_out))
+
+            for sol_tensor, ref_tensor in zip(out[:n_compare], ref_out[:n_compare]):
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)